extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1485,6 +1496,9 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.clearDescOverflowState_();
     }).then(() => this.exit_())
         .then(() => {
+          if (this.hasVerticalScrollbarWidth_) {
```

Comment:
> Change lightbox exit to only leave lightbox mode after the animation is complete when there is a scrollbar width. 

So the UX implication of this is that we won't be able to scroll until the animation is done. Do we have to make this exception based on whether or not the vertical scroll bar is present? Would it be better to keep this experience consistent and just move leave lightbox mode to post-animation regardless of vertical scrollbar? Otherwise we should probably add a comment explaining why.

Reply:
Added a comment. TL;DR want to leave earlier for those who can, since the blocked scrolling is annoying.